src/osp/Resource/Package.cpp

```diff
@@ -32,9 +32,18 @@ namespace osp
 Package::Package(std::string const& prefix, std::string const& packageName) :
```

Comment:
Most likely you want these constructor parameters to be passed in by value and then moved, not by const&.